### PR TITLE
Update preparation.md

### PR DESCRIPTION
**Pull request by Stefano1990**
added a little bit to mention that you need to install capacitor core and capacitor cli yourself.

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [x] Documentation
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch and _not_ the `master` branch
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] It's been tested on a Cordova (iOS, Android) app
- [ ] It's been tested on a Electron app
- [x] Any necessary documentation has been added or updated [in the docs](https://github.com/quasarframework/quasar/tree/dev/docs) (for faster update click on "Suggest an edit on GitHub" at bottom of page) or explained in the PR's description.

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

File: docs/src/pages/quasar-cli/developing-capacitor-apps/preparation.md [hunk 1]
```diff
@@ -59,7 +59,15 @@ In order to develop/build a Mobile app, we need to add the Capacitor mode to our
 $ quasar mode add capacitor
 ```
 
-## 3. Start Developing
+## 3. Install Capacitor
+
+Install Capacitor using npm ([Capacitor installation instructions](https://capacitorjs.com/docs/getting-started)):
+
+```bash
+$ npm install @capacitor/core @capacitor/cli
```
Comment (nurlansu) [inline docs/src/pages/quasar-cli/developing-capacitor-apps/preparation.md:67]: Why would someone need install them? They are already in package dependencies: see [here](https://github.com/quasarframework/quasar/blob/dev/app/templates/capacitor/package.json#L8-L9).

**Comment by Stefano1990** [comment]
That didn't actually work for me. I could add the extension and start it
and then error out.

**Comment by rstoenescu** [comment]
@Stefano1990 Can you put down the list of commands that you've used, please? Like @nurlansu mentioned, the src-capacitor/package.json has everything needed already.

**Comment by rstoenescu** [comment]
Closing due to lack of response.

**Pull request closed without merging**